freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/KerberosMgmtV1Service.java

```diff
@@ -20,6 +29,20 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(KerberosMgmtV1Service.class);
 
+    private static final String HOST_CREATION_FAILED = "Failed to create host";
+
+    private static final String SERVICE_PRINCIPAL_CREATION_FAILED = "Failed to create service principal";
+
+    private static final String SERVICE_ALLOW_FAILURE = "Request to allow the service to retrieve ket tab failed.";
+
+    private static final String KEYTAB_GENERATION_FAILED = "Failed to create keytab";
+
+    private static final String KEYTAB_FETCH_FAILED = "Failed to fetch keytab";
+
+    private static final String EMPTY_REALM = "Failed to create service as realm was empty.";
```

Comment:
nit: I noticed that some of the strings have a period at the end while others don't. It may be best to be consistent.